docs/fx-core/aad-help.md

```diff
@@ -150,4 +150,14 @@ Root cause of this error is that toolkit can not find Azure AD app with the same
 
 If you can find your Azure AD app, please check your network status and try again.
 
-If you can not find your Azure AD app, please check whether you logged in with the correct account. You can also remove objectId from env.default.json file and then try again.
\ No newline at end of file
+If you can not find your Azure AD app, please check whether you logged in with the correct account. You can also remove objectId from env.default.json file and then try again.
+
+## aad.Compliance
+To improve the security of app registration,please refer to [Best Practices for app registration](
```

Comment:
missing space before "please refer to...." and other nits:

To improve the security of app registration, please refer to [Best Practices for app registration](https://docs.microsoft.com/en-us/azure/active-directory/develop/security-best-practices-for-app-registration) for more detail.

Reply:
Fixed. Thanks.